gdsc-web-backend/Controllers/FaqController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using gdsc_web_backend.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace gdsc_web_backend.Controllers
+{
+    
+    [ApiController] 
+    [Route("api/[controller]")]
+    public class FaqController: ControllerBase
```

Comment:
Please rename FaqController to FaqsController, since a controller manages more faqs